content/blog/2017/2017-02-02-jenkins-community-survey

```diff
@@ -0,0 +1,32 @@
+---
+layout: post
+title: "The State of Jenkins - 2016 Community Survey "
+tags:
+- community
+- survey
+- continuous delivery
+- docker
+author: brao
+---
+
+[NOTE]
+====
+This is a guest post by Bhavani Rao, Marketing Manager at link:http://cloudbees.com[CloudBees] 
+====
+
+Last fall, prior to Jenkins World, CloudBees conducted a Community Survey on behalf of the Jenkins community. We received over 1200 responses  – and thanks to this input, we 
+have some interesting insights into what Jenkins users are doing.
+
+Based on the results, Jenkins is increasingly being used to support continuous delivery (CD). Jenkins 2, which introduced Pipeline-as-code and the Pipeline plugin as core to Jenkins, has skyrocketed to being adopted by nearly half Jenkins users.  Other data remained consistent with findings year to year. For example, the number of Jenkins users continues to increase, and 90% of survey respondents still consider Jenkins mission-critical. 
```

Comment:
Grammar: "by nearly half Jenkins users"

Use colon? "findings year to year. For example"